api/v1alpha3/machine_types.go

```diff
@@ -31,6 +31,12 @@ const (
 
 	// ExcludeNodeDrainingAnnotation annotation explicitly skips node draining if set
 	ExcludeNodeDrainingAnnotation = "machine.cluster.x-k8s.io.io/exclude-node-draining"
+
+	// MachineSetLabelName is the label set on machines if they're controlled by MachineSet
+	MachineSetLabelName = "cluster.x-k8s.io/set-name"
+
+	// MachineDeploymentLabelName is the label set on machines if they're controlled by MachineDeployment
```

Comment:
MachineDeployments don't manage machines. Is the idea that if we create a machine deployment the machines will have both MachineSet and MachineDeployment labels?

Reply:
yes, that is the idea.